LICENSE

```diff
@@ -1,3 +1,14 @@
+These patches consist of multiple software modules and patches for Oracle
+GraalVM. These patches are released under version 2 of the GNU General Public
+License with the “Classpath Exception.” The text of the foregoing licenses is
```

Comment:
I would put the "." after the quotes. Also these quotes are weirdly oblique.

Reply:
Apparently this is an American English difference! I'm happy to leave as is.

Reply:
Ok, even the dot inside the quotes??
I.e. `License with the “Classpath Exception.”` instead of `License with the “Classpath Exception”.`

Reply:
[Apparently](https://www.unr.edu/writing-speaking-center/student-resources/writing-speaking-resources/british-american-english#:~:text=Commas%20and%20Periods%20Within%20Quotation,is%20part%20of%20the%20quotation.)

Reply:
LOL